streams-util/src/main/java/org/apache/streams/util/SerializationUtil.java

```diff
@@ -49,7 +49,7 @@
      * @param serialized
      * @return
      */
-    public static Object deserialize(byte[] serialized, Class clazz) {
+    public static Object deserialize(byte[] serialized) {
```

Comment:
Changing the signature of a public method. Consider providing an overloaded variant with the original signature for compatibility?